.deploy/dev-gcp.json

```diff
@@ -1,5 +1,5 @@
 {
-  "ingresses": ["https://familie-dokument.dev.intern.nav.no", "https://familie-dokument.dev.nav.no"],
```

Comment:
For en som er glad i "prod-likt" -  burde vi ha `https://www-q0.nav.no/familie/dokument` som en ingress i preprod? 🤔

Reply:
Jeg er enig
Tidligere var det noe krøll(certificate) med dev.nav.no når man ikke brukte en subdomene under dev.nav.no, men kan prøve den på nytt og se om det virker fint.

Reply:
Det virker ikke. GCP støtter ikke dev.nav.no uten annen subdomene :/